test/includes/microcloud.sh

```diff
@@ -487,7 +487,6 @@ reset_snaps() {
     lxc exec "${name}" -- snap enable microcloud > /dev/null 2>&1 || true
     lxc exec "${name}" -- snap start lxd > /dev/null 2>&1 || true
     lxc exec "${name}" -- snap start microcloud > /dev/null 2>&1 || true
-    lxc exec "${name}" -- snap refresh lxd --cohort=+ || true
```

Comment:
This line was added in the same PR in https://github.com/canonical/microcloud/pull/300/commits/cf4746f0aca0fea510425e7a169f5d1e2a0c609b. Can you please squash those commits?

Reply:
Huh, that's odd. Must've been removed and my rebase added it back :)